src/interaction.ts

```diff
@@ -623,20 +566,13 @@ export class IndirectKeystoreInteraction extends KeystoreInteraction {
    * appropriate kind of `response` object and return the final result
    * of this interaction.
    *
-   * @param {Object} response - the raw response
-   * @returns {Object} the parsed response
-   *
    */
-  parse(response) {
+  parse(response: object) {
```

Comment:
My understanding is that `Record<string, any>` is preferable here but this is probably fine.

Reply:
A few things were extending this. I didn't want to touch it, but I had to to get the others typed for your above request.

Reply:
Changed in 35668a8805582ecc715ddc5e5214ac4de91c22ff